### PR TITLE
Add "Assert Silent" rule

### PR DESCRIPTION
**Pull request by fatkodima**
http://docs.seattlerb.org/minitest/Minitest/Assertions.html#method-i-assert_silent

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -269,6 +269,19 @@ assert_match expected, $stdout.read
...
 ----
 
+=== Assert Silent [[assert-silent]]
+
+Use `assert_silent` to assert that nothing was written to stdout and stderr.
+
+[source,ruby]
+----
+# bad
+assert_output('', '') { puts object.method }
```
Comment (koic) [inline README.adoc:279]: Can you update the following example method name?

```suggestion
assert_output('', '') { puts object.do_something }
```

File: README.adoc [hunk 2]
```diff
@@ -269,6 +269,19 @@ assert_match expected, $stdout.read
...
+
+# good
+assert_silent { puts object.method }
```
Comment (koic) [inline README.adoc:282]:
```suggestion
assert_silent { puts object.do_something }
```

**Comment by fatkodima** [comment]
Updated.

**Comment by koic** [comment]
👍

**Pull request merged**